HerPublicWebsite/Views/Questionnaire/CheckAnswers.cshtml

```diff
@@ -0,0 +1,206 @@
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using HerPublicWebsite.BusinessLogic.Models.Enums
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using HerPublicWebsite.Controllers
+@using System.Runtime.CompilerServices
+@using GovUkDesignSystem.Attributes
+
+@model HerPublicWebsite.Models.Questionnaire.CheckAnswersViewModel
+
+@{
+    ViewBag.Title = "Check your answers";
+}
+
+@section BeforeMain {
+    @await Html.GovUkBackLink(new BackLinkViewModel
+           {
+               Text = "Back",
+               Href = Model.BackLink,
+           })
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <form action="@Url.Action(nameof(QuestionnaireController.CheckAnswers_Post), "Questionnaire")" method="post" novalidate>
+            @(Html.HiddenFor(m => m.EntryPoint))
+            @Html.AntiForgeryToken()
+            
+            @{ var values = new Dictionary<string, object> { { "entryPoint", QuestionFlowStep.CheckAnswers } }; }
```

Comment:
I think there's been a misunderstanding of how this works. If you have a look at the SEA codebase you can see this in action.

The entry point is the page at which you re-enter the question flow. The point is to allow the back and forward link code to know where to go, especially if you have dependent questions (which is a much bigger thing on SEA).

On HER it would really only be the address flow at the moment - you want to be able to go back and forward through the postcode, address selection, etc pages but go back to the check your answers page either when you
- Click back on the first address page (which would be the entrypoint)
- Select an address from the list
- Select your local authority if you entered an address manually

Reply:
Ah I see I think I'm doing the right thing, but I've inverted the entrypoint to be where it's entered from I guess. I think it functions the same as far as I can tell, but it's not difficult to change it, it's just changing what entrypoint I pass through and check, but the logic stays the same I think